### ISSUE TITLE
Convergence of GAN

### ISSUE THREAD

**Issue by Amirhossein-Esmaili** [4thgen/DCGAN-CIFAR10#2]
Hello,

I am using [a PyTorch repository](https://github.com/ysbsb/dcgan-pytorch-cifar10) for training GANs for CIFAR-10 and changed it using your training tips (the architecture of G & D, the use of leaky ReLU and tanh activations, different learning rates for G and D, training G twice and D once in each iteration, etc.,). These have improved the results compared to when I used the PyTorch repository without any tuning.

However, the best training curve I get is as follows:

<img width="625" alt="Screen Shot 2021-12-17 at 11 37 26 AM" src="https://user-images.githubusercontent.com/33773340/146600915-11bf2911-1f54-4519-bfcd-53783335b021.png">

I think my results do not converge. The oscillation of loss values increases over time, rather than oscillation around a value with relatively low/constant. Do you have any idea or have encountered this phenomenon before?
